src/Resolving/NameServer.cs

```diff
@@ -96,20 +85,25 @@ public class NameServer : IResolver
         ///   If the question's domain does not exist, then the closest authority
         ///   (<see cref="SOARecord"/>) is added to the <see cref="Message.AuthorityRecords"/>.
         /// </remarks>
-        public async Task<Message> ResolveAsync(Question question, Message response = null, CancellationToken cancel = default(CancellationToken))
+        public Message Resolve(Question question, Message response = null)
         {
```

Comment:
This is a **breaking change**.  There are a number of projects already using this package.

Reply:
Returned method for back compatibility,
one notice IResolver does not provide such method, should it?